pkg/app/appnet/skywire_networker.go

```diff
@@ -50,7 +51,12 @@ func (r *SkywireNetworker) DialContext(ctx context.Context, addr Addr) (net.Conn
 		return nil, err
 	}
 
-	rg, err := r.r.DialRoutes(ctx, addr.PubKey, routing.Port(localPort), addr.Port, router.DefaultDialOptions())
+	rPK := addr.PubKey
+	if rPK.Null() {
+		return nil, fmt.Errorf("Empty public key: %s", rPK)
```

Comment:
Please change to this:
```golang
return nil, errors.New("cannot dial to address with empty public key")
```
- Golang errors should always start with a lowercase letter.
- If the public key is empty, we don't need to show it in the error message. It is empty after all.

Reply:
Right. Refactored the error.